src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
```

Comment:
Consider using a more descriptive class name that reflects its purpose or functionality.